stock_picking_report_valued/report/stock_picking_report_valued.xml

```diff
@@ -92,18 +90,9 @@
     <template
         id="valued_report_picking_has_serial_move_line"
         inherit_id="stock.stock_report_delivery_has_serial_move_line"
+        priority="17"
     >
-        <xpath expr="//td[@name='move_line_lot_qty_done']" position="after">
-            <t t-if="move_line.picking_id.state != 'done'">
-                    <td class="text-end">
-                        <span t-field="move_line.product_uom_qty" />
-                        <span t-field="move_line.product_uom_id" /></td>
-                </t>
-                <t t-elif="move_line.picking_id.state == 'done'">
-                    <td class="text-center">
-                        <span t-field="move_line.qty_done" />
-                        <span t-field="move_line.product_uom_id" /></td>
-                </t>
```

Comment:
Because the new columns are added to the table `stock_move_line_table` which shown when o.state = done, so, we don't need these lines of code.